### PR TITLE
compose with redux dev tools in development mode only

### PR DESCRIPTION
**Pull request by teleaziz**

### CODE REVIEW

File: src/state/create-store.js [hunk 1]
```diff
@@ -20,8 +20,12 @@ import type { Store } from './store-types';
 // We are checking if window is available before using it.
 // This is needed for universal apps that render the component server side.
 // Details: https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup
+const isDevelopment: boolean = process.env.NODE_ENV === 'development';
+
 const composeEnhancers =
-  typeof window !== 'undefined' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
+  isDevelopment &&
```
Comment (alexreardon) [inline src/state/create-store.js]: `process.env.NODE_ENV !== 'production'`

Sometimes there are other env strings used that are not development such as 'test'

File: src/state/create-store.js [hunk 2]
```diff
@@ -20,8 +20,12 @@ import type { Store } from './store-types';
 // We are checking if window is available before using it.
 // This is needed for universal apps that render the component server side.
 // Details: https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup
+const isDevelopment: boolean = process.env.NODE_ENV === 'development';
```
Comment (alexreardon) [inline src/state/create-store.js]:
```suggestion

```

File: src/state/create-store.js (hunk 1 above)
Comment (alexreardon) [inline src/state/create-store.js]:
```suggestion
  process.env.NODE_ENV !== 'production' &&
```

**Comment by alexreardon** [comment]
Thanks!!!

**Comment by alexreardon** [comment]
This will go out in the next release (some time in the next two weeks 🤞 )

**Comment by teleaziz** [comment]
Thanks @alexreardon for taking a look, great work on this package btw 🥇

**Pull request merged**